src/shared/pretty-print.h

```diff
@@ -16,10 +16,11 @@ int terminal_urlify_man(const char *page, const char *section, char **ret);
 
 typedef enum CatFlags {
         CAT_FLAGS_MAIN_FILE_OPTIONAL = 1 << 0,
+        CAT_FLAGS_SUPPRESS_COMMENTS  = 1 << 1,
```

Comment:
it's kinda weird having the plural "flags" in the individual flag names. I think we should remove it, usually if an enum type is call FooBar we tend to call the enum values FOO_XYZ and not FOO_BAR_XYZ anyway. In particular in this case that makes a ton of sense.